internal/cluster/common.go

```diff
@@ -66,6 +66,25 @@ func updateClusterStatus(log logrus.FieldLogger, db *gorm.DB, clusterId strfmt.U
 	return cluster, nil
 }
 
+func updateClusterProgress(log logrus.FieldLogger, db *gorm.DB, clusterId strfmt.UUID, progress string,
+	extra ...interface{}) (*common.Cluster, error) {
+	var cluster *common.Cluster
+	var err error
+
+	extra = append(append(make([]interface{}, 0), "progress_progress_info", progress), extra...)
+
+	now := strfmt.DateTime(time.Now())
+	extra = append(extra, "progress_progress_updated_at", now)
+
+	if cluster, err = UpdateCluster(log, db, clusterId, "installing", extra...); err != nil {
```

Comment:
```suggestion
	if cluster, err = UpdateCluster(log, db, clusterId, models.HostStageInstalling, extra...); err != nil {
```

Reply:
Note that models.HostStageInstalling is "Installing" with a capital I, so this probably didn't work before.  Need to add tests.

Reply:
Following the additional discussion I had with Avishay, we'll use [models.ClusterStatusInstalling](https://github.com/openshift/assisted-service/blob/92e3dada0586054039cb53f468635c1b3501f01a/models/cluster.go#L653).

Another problem here is that [UpdateCluster() can retrieve](https://github.com/openshift/assisted-service/blob/92e3dada0586054039cb53f468635c1b3501f01a/internal/cluster/common.go#L92) the cluster from DB regardless of what it gets in srcStatus.

[We do have tests for this](https://github.com/openshift/assisted-service/blob/e6a1f1b48aa83bbd62bab6a6ee966cdb0bee5fd2/internal/cluster/common_test.go#L42-L76), but for some reason, this still happens.

I discussed this with Yuval, and we'll take a closer look.
However, IMO a fix for that should be done in a separate PR.

Reply:
Why not use UpdateCluster() though?  It doesn't seem like the bug would affect your code.

Reply:
I do use `UpdateCluster()`, as you proposed.
and the with status retrieval turned out to be just that I had an old image deployed.
When updated - works as expected.